sentry/src/main/java/io/sentry/config/Lookup.java

```diff
@@ -4,23 +4,79 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
 /**
  * Handle lookup of configuration keys by trying JNDI, System Environment, and Java System Properties.
  */
 public final class Lookup {
```

Comment:
Changed from a static utility class to a regular one so that the config file can be found/parsed once at creation. We could do this statically but it wouldn't be very testable if it were static state.